app/app.go

```diff
@@ -158,7 +165,7 @@ var (
 	// https://github.com/CosmWasm/wasmd/blob/02a54d33ff2c064f3539ae12d75d027d9c665f05/x/wasm/internal/types/proposal.go#L28-L34
 	EnableSpecificProposals = ""
 
-	Upgrades = []upgrades.Upgrade{v2.Upgrade, v2_2_5.Upgrade}
+	Upgrades = []upgrades.Upgrade{v2.Upgrade, v2_2_5.Upgrade, v3.Upgrade}
```

Comment:
@hoangdv2429 - is there a reason we are calling these old upgrades?  I think they are no longer needed.

Reply:
No, I think we can remove old upgrade calls

Reply:
OK, please remove them :)

Reply:
Should I make a PR to this PR ? cc: @faddat